packages/zoe/tools/fakePriceAuthority.js

```diff
@@ -14,12 +14,14 @@ import { natSafeMath } from '../src/contractSupport/index.js';
 import './types.js';
 import '../exported.js';
 
+/** @typedef {number | bigint} Bigish */
```

Comment:
this seems like a recipe for sloppiness. please consider removing.

If it kept, there must be a better name. When read out of context `Bigish` conveys nothing about what is big (there's no mention of number or int).

Reply:
Nat will coerce either, I think. Any reason not to require the caller to do that conversion?